src/main/java/com/vinylteam/vinyl/util/ControllerResponseUtils.java

```diff
@@ -15,21 +15,21 @@
     public static UserSecurityResponse getResponseFromMap(Map<String, Object> responseMap){
         UserSecurityResponse response = new UserSecurityResponse();
         response.setMessage((String)responseMap.get("message"));
-        response.setResultCode((String)responseMap.get("resultCode"));
         response.setToken((String)responseMap.get("token"));
         response.setUser((UserDto)responseMap.get("user"));
         return response;
     }
 
     public static Map<String, String> getStatusInfoMap(String code, String message) {
-        return Map.of(
-                "resultCode", code,
-                "message", message);
+        return Map.of("message", message);
+    }
+
+    public static UserSecurityResponse getResponseWithMessage(String message) {
+        return setStatusInfo(new UserSecurityResponse(), null, message);
     }
 
     public static UserSecurityResponse setStatusInfo(UserSecurityResponse response, String code, String message) {
         response.setMessage(message);
-        response.setResultCode(code);
         return response;
     }
 
```

Comment:
line 43-43
please make it one line
return Map.of("user", userMapper.mapUserToDto(user), "token", token);